### PR TITLE
[RDR] [UI] Text needs correction for Relocate operation

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
fixes: https://issues.redhat.com/browse/OCSBZM-4777
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2133775

image
image

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
UX is working on the text

File: packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx [hunk 1]
```diff
@@ -207,7 +207,7 @@ export const RelocateModal: React.FC<ModalProps> = (props) => {
       {...props}
       title={t('Relocate application')}
       description={t(
-        'All changes in system workloads and their available resources will be transferred back to the origin (primary) cluster.'
+        'All system workloads and their available resources will be transferred to the preferred cluster.'
```
[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx:210]: @Chandan141996 any updates on this ??

[the remark this answers is not in the document]
Comment (Chandan141996) [reply packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx:210]: Hey @SanjalKatiyar @GowthamShanmugam 

You may use these new descriptions provided for the following terminologies

**Relocate** Relocating terminates your application on its current cluster, syncs its most recent snapshot to the selected target cluster, and then brings up your application.

**Failover** Failing over force stops active replication and deploys your application on the selected target cluster. Recommended only when the primary cluster is down.

[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx:210]: @Chandan141996 thanks for the suggestion...
@debjyoti-pandit plz update the PR accordingly...

File: packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx [hunk 2]
```diff
@@ -191,7 +191,7 @@ export const FailoverModal: React.FC<ModalProps> = (props) => {
       {...props}
       title={t('Failover application')}
       description={t(
-        'All system workloads and their available resources will be transferred to the recovery (failover) cluster.'
+        'Failover Failing over force stops active replication and deploys your application on the selected target cluster. Recommended only when the primary cluster is down.'
```
Comment (GowthamShanmugam) [inline packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx:194]: "Failover Failing over "?
 this is confusing 
 
 you mean:
 
 Failover force stops active replication and deploys your application on the selected target cluster. Recommended only when the primary cluster is down.?

Comment (debjyoti-pandit) [reply packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx:194]: ack

File: packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx [hunk 3]
```diff
@@ -207,7 +207,7 @@ export const RelocateModal: React.FC<ModalProps> = (props) => {
       {...props}
       title={t('Relocate application')}
       description={t(
-        'All changes in system workloads and their available resources will be transferred back to the origin (primary) cluster.'
+        'Relocate Relocating terminates your application on its current cluster, syncs its most recent snapshot to the selected target cluster, and then brings up your application.'
```
Comment (GowthamShanmugam) [inline packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal.tsx:210]: Same here
you mean:
          Relocate terminates your application on its current cluster, syncs its most recent snapshot to the selected target cluster, and then brings up your application. ?

**Pull request merged**